build/circle-test.sh

```diff
@@ -2,8 +2,6 @@
 
 set -euo pipefail
 
-export SKIP_BOOTSTRAP=1
```

Comment:
Is there a way to set an environment variable in a docker container when the container is built? That would be more robust than this approach.